contactsummary/js/dist/contact-summary.min.js

```diff
@@ -1 +1,1763 @@
ontroller("SicknessCtrl",["$log","LeaveService",t])}),define("contact-summary/modules/directives",["common/angular"],function(e){"use strict";return e.module("contactsummary.directives",[])}),define("contact-summary/directives/donutChart",["common/d3","contact-summary/modules/directives"],function(e,t){"use strict";function n(e){this.height=this.width=e[0].clientWidth,this.radius=this.width/2||60,this.thickness=this.thickness||15}function r(){return e.svg.arc().innerRadius(this.radius-this.thickness).outerRadius(this.radius)}function i(t,n,r){var i=e.scale.category20();return t.selectAll("path").data(r).enter().append("path").attr("fill",function(e,t){return i(t)}).attr("class",function(e,t){return"chart-color-"+t}).attr("d",n)}function s(){var t=e.layout.pie().sort(null).value(function(e){return e.value[this.itemKey]}.bind(this));return t(e.entries(this.items))}function o(t){return e.select(t).append("svg").attr("width",this.width).attr("height",this.height).append("g").attr("transform","translate("+this.width/2+","+this.height/2+")")}t.directive("csDonutChart",["$log",function(e){return e.debug("Directive: csDonutChart"),{controllerAs:"CsDonutChartCtrl",restrict:"AE",scope:{radius:"@",thickness:"@",items:"=",itemKey:"@",ready:"="},controller:["$scope","$element",function(e,t){this.drawChart=function(){n.call(angular.extend(this,e),t),i(o.call(this,t[0]),r.call(this),s.call(this))}}],link:function(e,t,n,r){var i=e.$watch(function(){return e.ready},function(e,t){e===!0&&(r.drawChart(),i())})}}}])}),define("contact-summary/app",["common/angular","contact-summary/modules/filters","contact-summary/modules/services","contact-summary/modules/settings","contact-summary/controllers/contactSummary","contact-summary/controllers/keyDates","contact-summary/controllers/keyDetails","contact-summary/controllers/leave","contact-summary/controllers/sickness","contact-summary/directives/donutChart"],function(e){var t=e.module("contactsummary",["ngRoute","ngResource","ui.bootstrap","contactsummary.controllers","contactsummary.directives","contactsummary.filters","contactsummary.services","contactsummary.settings"]);t.config(["settings","$routeProvider","$resourceProvider","$httpProvider","$logProvider",function(e,t,n,r,i){i.debugEnabled(e.debug),t.when("/",{controller:"ContactSummaryCtrl",controllerAs:"ContactSummaryCtrl",templateUrl:e.pathBaseUrl+e.pathTpl+"mainTemplate.html",resolve:{}}).otherwise({redirectTo:"/"}),n.defaults.stripTrailingSlashes=!1,r.defaults.headers.common["X-Requested-With"]="XMLHttpRequest"}]),t.run(["settings","$rootScope","$q","$log",function(e,t,n,r){r.debug("app.run"),t.pathTpl=e.pathTpl,t.prefix=e.classNamePrefix}])}),require.config({urlArgs:"bust="+(new Date).getTime(),paths:{"contact-summary":CRM.vars.contactsummary.baseURL+"/js/src/contact-summary"}}),require(["contact-summary/app"],function(){document.addEventListener("contactsummaryLoad",function(){angular.bootstrap(document.getElementById("contactsummary"),["contactsummary"])})});
```

Comment:
Yeah, it turns out that the file _contactsummary/js/src/contact-summary/controllers/contactSummary.js_ was missing an empty line at the end. This is also fixed now